zeus/util/metric.py

```diff
@@ -37,8 +37,23 @@ def zeus_cost(energy: float, time: float, eta_knob: float, max_power: int) -> fl
 
 
 class ZeusCostThresholdExceededException(Exception):
-    """
-    Raised when the predicted cost of the next epoch exceeds the cost threshold.
+    """Raised when the predicted cost of the next epoch exceeds the cost threshold.
+    This exception is used for terminating all the processes when doing data
+    parallel training.
+
+    When doing data parallel training on multiple processes, ONLY the master
```

Comment:
```suggestion

    This exception is used for terminating all the processes when doing data
    parallel training with multiple processes, because ONLY the master
```